core/src/main/java/com/netflix/iceberg/PropertiesUpdate.java

```diff
@@ -68,6 +68,12 @@ public UpdateProperties remove(String key) {
     return this;
   }
 
+  @Override
+  public UpdateProperties format(FileFormat format) {
```

Comment:
I like adding the method, but it should be clear that this is the default file format for the table. Tables, by design, can contain multiple file formats so you can change from one format to another and so you can write from a streaming system to Avro and then compact to a long-term storage format later.

Maybe this should be `preferredFormat` or `defaultFormat`?

Reply:
Let's go with defaultFormat.